scripts/utils/trading_strategy_helpers.js

```diff
@@ -375,7 +375,20 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
    * @param {string} functionName Name of the function that is to be executed (can be "requestWithdraw" or "withdraw")
    * @returns {Transaction} Multisend transaction to be sent from masterAddress for withdraw requests or claims
    */
-  const buildGenericFundMovement = async function (masterAddress, withdrawals, functionName) {
+  const buildGenericFundMovement = async function () {
+    return buildBundledTransaction(await transactionGenericFundMovement(...arguments))
+  }
+
+  /**
+   * Creates transactions for a collection of operations (either withdraw or requestWithdraw) on BatchExchange
+   * on behalf of a fleet of brackets owned by a single "Master Safe"
+   *
+   * @param {Address} masterAddress Ethereum address of Master Gnosis Safe (Multi-Sig)
+   * @param {Withdrawal[]} withdrawals List of {@link Withdrawal} that are to be bundled together
+   * @param {string} functionName Name of the function that is to be executed (can be "requestWithdraw" or "withdraw")
+   * @returns {Transaction} Multisend transaction to be sent from masterAddress for withdraw requests or claims
+   */
+  const transactionGenericFundMovement = async function (masterAddress, withdrawals, functionName) {
```

Comment:
I think `buildGenericFundMovement` is never used anywhere now, `transactionGenericFundMovement` should be enough.